index.html

```diff
@@ -0,0 +1,141 @@
+        </div>
+        <div id="featured-speakers">
+            <!-- <h4>Featured Speakers</h4>
+            <div class="about-speakers">
+                <img class="face" src="images/benkler.png" alt="Yochai Benkler's face/picture">
+                <h3>Yochai Benkler</h3>
+                <p class="pink">Berkmen Professor of Enterpreneurial Legal Studies at Harvard Law School</p>
+                <p>Benker studies commons-based peer production, and published his seminal book. The Wealth of Networks in 2006.</p>
+            </div>
+            <div class="about-speakers">
+                <img class="face" src="images/Noh.png" alt="SohYeong Noh's face/picture">
+                <h3>SohYeong Noh</h3>
+                <p class="pink">Director of Art Centre Nabi and a board member of CC Korea</p>
+                <p>As the main venue for new media art production in Korea, Nabi promotes cross-disciplinary collaboration and understands among science technology, humanities, and the arts.</p>
+            </div>
+            <div class="about-speakers">
+                <img class="face" src="images/tretikov.png" alt="Lila Tretikov's face/picture">
+                <h3>Lila Tretikov</h3>
+                <p class="pink">Executive Director of the Wikimedia Foundation</p>
+                <p>Lila Tretikov is the Executive Director of the Wikimedia Foundation, the nonprofit organization that operates Wikipedia. Wikipedia is freely available in 290 languages and used by nearly half a billion people around the world every month.</p>
+            </div>
+            <div class="about-speakers">
+                <img class="face" src="images/chon.png" alt="Kilnam Chon's face/picture">
+                <h3>Kilnam Chon</h3>
+                <p>Kilnam Chon helped bring the Internet to Asia and is an outspoken advocate for the open web and digital commons. In 2012, he was inducted into the inagural class of the Inernet Society's (ISOC) Internet Hall of Fame.</p>
+            </div>
+            <div class="about-speakers">
+                <img class="face" src="images/reda.png" alt="Julia Leda face/picture">
+                <h3>Julia Leda</h3>
+                <p class="pink">President of Young Pirates of Europe</p>
+                <p>European ingetration, political democracy and participaiton of youth through online as her major condern, Reda's report outlining potential changes to EU copyright law was approved by the Parliament in July.</p>
+            </div>
+            <div class="about-speakers">
+                <img class="face" src="images/merkley.png" alt="Ryan Merkley face/picture">
+                <h3>Ryan Merkley</h3>
+                <p class="pink">CEO of Creative Commons, ex COO of the Mozilla Foundation</p>
+                <p>Ryan had been leading open-source projects at the Mozilla Foundation such as the open source movement.</p>
+            </div> -->
+        </div>
+        <div class="partners">
```

Comment:
Kindly use semantic section element instead of div for making your web page more SEO friendly.